src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -358,6 +358,17 @@ public function everyThirtyMinutes()
     }
 
     /**
+     * Schedule the event to run every a random number of minutes.
+     * @param  int $min
+     * @param  int $max
+     * @return $this
+     */
+    public function everyRandomMinutes($min, $max)
```

Comment:
Pretty good idea. I think it would be better if default values will be 1 and 59. I'll do it in a minute.